src/schema/manifest-schema.json

```diff
@@ -150,6 +150,47 @@
         "author": {
             "description": "Legacy field from Chrome. Use \"developer\" instead.",
             "type": "string"
+        },
+        "commands": {
+            "type": "object",
+            "additionalProperties": {
+                "type": "object",
+                "properties": {
+                    "suggested_key": {
+                        "type": "object",
+                        "properties": {
+                            "default": {
+                                "$ref": "#/definitions/KeyName"
+                            },
+                            "mac": {
+                                "$ref": "#/definitions/KeyName"
+                            },
+                            "linux": {
+                                "$ref": "#/definitions/KeyName"
+                            },
+                            "windows": {
+                                "$ref": "#/definitions/KeyName"
+                            },
+                            "chromeos": {
+                                "type": "string"
+                            },
+                            "android": {
+                                "type": "string"
+                            },
+                            "ios": {
+                                "type": "string"
+                            },
+                            "additionalProperties": {
```

Comment:
This is how the Firefox schema is set up but I think it's actually wrong, unfortunately. I don't think it's hurting anything but `additionalProperties` should be at the same level as `properties`.